packages/block-editor/src/components/block-variation-transforms/style.scss

```diff
@@ -1,38 +1,4 @@
 .block-editor-block-variation-transforms {
+	display: flex;
 	padding: 0 $grid-unit-20 $grid-unit-20 52px;
-	width: 100%;
-
-	.components-dropdown-menu__toggle {
```

Comment:
Thanks for the follow-ups!

Are these style removals intentional? Most of these appear to affect the dropdown list version of the list of variations, [which is used](https://github.com/WordPress/gutenberg/blob/33d84b036592a5bf31af05b7710f3b2b14163dc4/packages/block-editor/src/components/block-variation-transforms/index.js#L146) when not all of the variations have unique icons.

To test this, with the changes applied to scope the social icons for `transform`, in `packages/block-library/src/social-link/variations.js` also set any of the social icon blocks to use the same icon as one of the other variations. E.g. set any of them to use the `YouTubeIcon` icon.

Here's a quick view of trunk versus this PR:

| Trunk | This PR |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/14988353/200944344-6c4723ec-07c5-4ff0-a95a-59d565549947.png) | ![image](https://user-images.githubusercontent.com/14988353/200944189-8642ff3a-e98e-4b4d-a025-68ebe2d83567.png) |

Reply:
Oops, I need more coffee. I forgot to push the changes 😄 Thanks for your patience!

Reply:
> Oops, I need more coffee. 

No worries, I have the opposite problem of having drunk too much coffee!

I've re-pulled the changes, and it looks like there are still some differences in the dropdown popover for the transform to variation list. Visually it looks like there's now additional padding / a larger width of the popover, and it overflows the area so there's a horizontal scrollbar:

| Trunk | This PR |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/14988353/200948398-350425d7-3704-4b40-a666-6339d5ef3b62.png) | ![image](https://user-images.githubusercontent.com/14988353/200948445-b16dbbc2-b4a6-4002-9e57-30edeaebb90f.png) |